tests/test_server.py

```diff
@@ -57,7 +57,23 @@ def udev_rules(self):
             rules.unlink()
             self.reload_udev_rules()
 
-    @pytest.fixture(autouse=True)
+    @pytest.fixture(autouse=True, scope='session')
+    def server(self):
+        p = None
+        logs = f'ratbag-emu-log-{strftime("%Y-%m-%d_%H-%M")}'
+        with open(f'{logs}-stdout.txt', 'w') as stdout, \
+             open(f'{logs}-stderr.txt', 'w') as stderr:
```

Comment:
Is there any way for this files to be recovered from the CI?

Reply:
sure, I can add those as artifacts.

On this note, I should rewrite the whole `libratabg-tests` history, but I'll be out for the next 2-3 weeks in conferences, so I wanted to get the PR in first.

Reply:
Do you think we can get this working before that? Both in here and in libratbag?

Reply:
well, too late, I already pushed the button.

I could do it, but I really don't have time right now. I'll try to have it by the end of the day however.

Reply:
No, I meant do you think we can get the CI working, here and in libratbag, before you leave? If you don't have time I can try to port this to libratbag, I don't think it should be that hard.

Reply:
Ci is already passing for this repo.
The setup has been done so that if the only thing we need is install hid-tools, install requirements.txt, run pytest it should be working for libratbag.

However, there is a chance we will need to manually install ratbag-emu for libratbag, so there might be tweaks to do